templates/android/template/app/build.gradle

```diff
@@ -84,6 +92,13 @@ android {
             renderscriptDebuggable true
             // resValue  "string", "app_name", "${PROP_APP_NAME}-dbg"
             // applicationIdSuffix ".debug"
+
+            externalNativeBuild {
+                cmake {
+                    cFlags "-fvisibility=default -fno-omit-frame-pointer"
+                    cppFlags "-fvisibility=default -fno-omit-frame-pointer"
```

Comment:
Is it better to make visibility hidden in debug mode so that symbols missing errors would show at the dev stage?

Reply:
It should not be hidden in debug mode since if it is hidden and there is a crash happen, the crash stack in Android Studio will be function address only (without the function name).